Reading List/Reading List/BookController.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  BookController.swift
+//  Reading List
+//
+//  Created by Cameron Dunn on 1/17/19.
+//  Copyright © 2019 Lambda School. All rights reserved.
+//
+
+import Foundation
+
+class BookController {
+    var books : [Book] = []
+    var readBooks : [Book]{
+        let booksThatHaveBeenRead = books.filter{$0.hasBeenRead == true}
+        return booksThatHaveBeenRead
+    }
+    var unreadBooks : [Book]{
+        let booksThatHaventBeenRead = books.filter{$0.hasBeenRead == false}
+        return booksThatHaventBeenRead
+    }
+    var readingListURL : URL?{
+        guard let documentsDirectory = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first else {return nil}
+        let finalLocation = documentsDirectory.appendingPathComponent("ReadingList.plist")
+        return finalLocation
+    }
+    func create(title: String, reasonToRead reason: String){
+        var book = Book()
+        book.reasonToRead = reason
+        book.title = title
+        books.append(book)
+        saveToPersistentStore()
+    }
+    func delete(bookToDelete book: Book){
+        let index = books.index(of: book)
+        books.remove(at: index!)
```

Comment:
I would suggest you try to avoid force unwrapping, it's not a good practice. In fact optional one of main differences from other languages, because you're checking values to avoid crashes. Check [this](https://blog.timac.org/2017/0628-swift-banning-force-unwrapping-optionals/) out 
```suggestion
        guard let index = books.index(of: book) else { return }
        books.remove(at: index)
```